.changeset/sharp-phones-check.md

```diff
@@ -0,0 +1,6 @@
+---
+'@contentful/f36-component': minor
+'@contentful/f36-utils': minor
```

Comment:
@denkristoffer Feel free to change to `patch`. I think it should be a minor version bump, but not sure how you handle this for Forma